src/app/pages/MainArticlesScreen/components/ArticleComponent/index.ts

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { injectIntl } from 'react-intl';
+
+import ArticleComponent from './Article.component';
+
+export const Article = compose(
+  injectIntl,
+  connect(),
```

Comment:
empty connect?

Reply:
I left it based on our create-web-app (was it left to make it easier to extend later?). 
Should I delete it?